spec/javascripts/models/beta/notification_spec.js

```diff
@@ -0,0 +1,48 @@
+import * as Notification from 'models/beta/notification.js';
+
+describe('Notification model', () => {
+  describe('createNotification', () => {
+    it('creates a new notification with an id and the same message/type', () => {
+      const myNotification = { message: 'message', type: 'error' };
+
+      const newNotification = Notification.createNotification(myNotification);
+
+      expect(newNotification.id).not.toBeUndefined();
+      expect(newNotification.message).toEqual(myNotification.message);
+      expect(newNotification.type).toEqual(myNotification.type);
+    });
+  });
+
+  describe('addNotification', () => {
+    it('returns an array with the new notification', () => {
+      const notifications = [
+        { id: 1 },
+        { id: 2 }
+      ];
+
+      const newNotification = { id: 3 };
```

Comment:
We should test the scenario for when the notification is an array instead of a single item.